### PR TITLE
More Oracle integration tests

### PR DESCRIPTION
**Pull request by aalbu**

### CODE REVIEW

File: presto-oracle/src/test/java/io/prestosql/plugin/oracle/BaseOracleIntegrationSmokeTest.java [hunk 1]
```diff
@@ -90,9 +99,44 @@ public void testShowCreateTable()
...
+        assertTrue(getQueryRunner().tableExists(getSession(), tableName));
+
+        assertUpdate("DROP TABLE " + tableName);
+        assertFalse(getQueryRunner().tableExists(getSession(), tableName));
+    }
+
+    @Test
+    public void testInsert()
+    {
+        try (TestTable table = new TestTable(onOracle(), getUser() + ".test_insert", "(x number(19), y varchar(100))")) {
```
Comment (findepi) [inline presto-oracle/src/test/java/io/prestosql/plugin/oracle/BaseOracleIntegrationSmokeTest.java:116]: `io.prestosql.testing.AbstractTestDistributedQueries#testInsert` is run for Oracle as well

File: presto-oracle/src/test/java/io/prestosql/plugin/oracle/BaseOracleIntegrationSmokeTest.java [hunk 2]
```diff
@@ -113,39 +157,65 @@ public void testPredicatePushdown()
...
+        predicatePushdownTest("float(126)", "123456789.987654321", "<=", "CAST(123456789.99 AS DOUBLE)");
+        predicatePushdownTest("BINARY_FLOAT", "5.0f", "=", "CAST(5.0 AS REAL)");
+        predicatePushdownTest("BINARY_DOUBLE", "20.233", "=", "CAST(20.233 AS DOUBLE)");
+        predicatePushdownTest("NUMBER(5,3)", "5.0", "=", "CAST(5.0 AS DECIMAL(5,3))");
     }
 
-    private void predicatePushdownTest(String oracleType, String oracleLiteral, String operator, String filterLiteral, String compareLiteral)
-            throws Exception
+    @Test
+    public void testPredicatePushdownForChars()
```
Comment (findepi) [inline presto-oracle/src/test/java/io/prestosql/plugin/oracle/BaseOracleIntegrationSmokeTest.java:181]: is it just method split?

wonder whether this is covered by `io.prestosql.testing.AbstractTestDistributedQueries#testDataMappingSmokeTest`

(perhaps not full?)

Comment (aalbu) [reply presto-oracle/src/test/java/io/prestosql/plugin/oracle/BaseOracleIntegrationSmokeTest.java:181]: There is some overlap, but this method additionally checks that predicates are pushed dow in the presence of casts.

Comment (findepi) [reply presto-oracle/src/test/java/io/prestosql/plugin/oracle/BaseOracleIntegrationSmokeTest.java:181]: oh, i missed that -- the method tests that the PPD _actually_ takes place, not only that results are correct

**Review by losipiuk — COMMENTED** [review]
mostly fine. One comment

**Comment by losipiuk** [comment]
Flaky: https://github.com/prestosql/presto/issues/4973

**Comment by losipiuk** [comment]
The other failure is due to fact this PR is based on broken master before https://github.com/prestosql/presto/pull/5211

**Pull request merged**